.github/workflows/foundation-ang-trigger.yml

```diff
@@ -0,0 +1,53 @@
+# Controls when the action will run. The workflow runs when manually triggered using the UI or API
+
+name: Foundation_Ang_On_Demand
+
+on:
+  push:
+    branches:
+      - dev/angular13
+      - dev/angular12
+  workflow_dispatch:
+    # Inputs the workflow accepts.
+    inputs:
+      foundation_ang_version:
+        description: 'Foundation-ang version'
+        required: true
+        default: '7.0.0'
```

Comment:
How can parameter be both required and have default value?

Reply:
The workflow_dispatch properties can have different types like string or boolean.
To run the workflow, it needs to have foundation_ang_version as a string (7.0.0 in this case). So if it's empty it's informed and doesn't allow running workflow without it. It means it's mandatory to have this value to run.

Reply:
As I understand the `default` value will be put into the form when you trigger the workflow (e.g **Log level**):

<img width="268" alt="image" src="https://user-images.githubusercontent.com/1961590/195295339-777a9e89-9df7-4652-a0d4-3438b62342ca.png">

But I agree that we can remove it because it is always required to enter by the developer.

Reply:
That works similarly to the forms input. So, because it is part of the workflow to overwrite the foundation-ang version, it should be mandatory so we need to keep it.

Reply:
ok thanks.
I would suggest then to remove both required and default.
So if it is not set, you `npm info` to get the latest version and use it

Reply:
what are the benefit of not having required and default

Reply:
Most of the time, I would consider the first and second options, although, the third option is more or less the same as the second. The use case of it is having a bug from/in a specific version of the libraries and it gives us the capability to debug it better. 
On top of that, based on [github-workflow.json](url) , we can't remove `required`, It is mandatory to have it. I will turn it to false and I will remove the default value.